generators/app/templates/src/utils/arrayUtils.js

```diff
@@ -9,5 +9,5 @@ export function stringArrayToObject(actionsArray, namespace = '') {
   if (actionsArray.some(actionName => !actionName || typeof actionName !== 'string')) {
     throw new Error('Action names must be strings and must not be empty');
   }
-  return Immutable(actionsArray).asObject(actionName => [actionName, `${namespace}:${actionName}`]);
+  return Immutable(actionsArray).asObject(actionName => [actionName, `${namespace}${actionName}`]);
```

Comment:
Shouldn't this be
```
namespace ? `${namespace}/${actionName}` : `${actionName}`
```
?

Reply:
In sura we changed it to 
```
return Immutable(actionsArray).asObject(actionName => [
    actionName,
    `${namespace}${namespace ? ':' : ''}${actionName}`
  ]);
```

Reply:
we could use the slash to separate the namespace and the action name! is `namespace/actionName` clearer than `namespace:actionName`?

@sbalay @mvbattan @guidoprinc

Reply:
We are also using this for i18n declarations. I think that the `colon` is clearer but what you guys prefer is better.

Reply:
IMHO using a slash will keep consistency across action names (e.g. those coming from redux form).
On the other hand, a colon will help to distinguish between our actions and other libraries actions.